packages/packages/api/src/medical/models/link.ts

```diff
@@ -0,0 +1,19 @@
+import { Patient } from "./patient";
+
+export enum LinkSource {
+  commonWell = "CommonWell",
+  careQuality = "CareQuality",
+  eHealthExchange = "eHealthExchange",
+}
+export type PatientLinks = {
```

Comment:
Can we `TODO` reusing the API schemas and adding a reference to that ticket number on these schemas, please?